cfpb/consumerfinance.gov
# Find a Housing Counselor coronavirus sidebar

Adds coronavirus sidebar breakout to find a housing counselor page.

## Additions

- New sidebar on FAHC


## How to test this PR

1. Check out branch, then navigate to [FAHC](http://localhost:8000/find-a-housing-counselor/) and verify new sidebar


## Screenshots

<img width="1634" alt="Screen Shot 2021-02-01 at 11 53 11 AM" src="https://user-images.githubusercontent.com/778171/106511895-bda3f900-6485-11eb-9e89-fd921f05039d.png">


## Notes
- I elected not to use the sidebar-breakout include because it seemed like formatting all the content to be passed might be overly complicated, but I'm happy to reconsider if that seems preferable.
